crates/iota/src/name_commands.rs

```diff
@@ -1311,6 +1854,30 @@ async fn fetch_renewal_config(context: &mut WalletContext) -> anyhow::Result<Ren
     Ok(entry.renewal_config)
 }
 
+async fn handle_transaction_result<Fun, F>(
+    res: IotaClientCommandResult,
+    verbose: bool,
+    fun: Fun,
+) -> anyhow::Result<NameCommandResult>
+where
+    Fun: FnOnce(IotaTransactionBlockResponse) -> F,
+    F: futures::Future<Output = anyhow::Result<NameCommandResult>>,
+{
+    if verbose {
+        println!("{res}");
```

Comment:
```suggestion
        println!("{res}\n");
```
Easier to differentiate both outputs. Not sure if that's the best place to put it though, I'll leave it to you.